RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h

```diff
@@ -17,7 +18,7 @@ namespace edm {
 namespace hgcal {
   class RecHitTools {
   public:
-  RecHitTools() : geom_(nullptr), fhOffset_(0), bhOffset_(0) {}
+    RecHitTools(int mode=0) : mode_(mode), geom_(nullptr), fhOffset_(0), bhOffset_(0) {}
```

Comment:
IIUC, this should now allow syntax like ```RecHitTools rh = 42;```

Where is this constructor with the int argument used?
It would be more readable to me to introduce a setMode method.